cylc/flow/cfgspec/workflow.py

```diff
@@ -2195,7 +2184,7 @@ def upgrade_graph_section(cfg: Dict[str, Any], descr: str) -> None:
                         keys.add(key)
                 if keys and not cylc.flow.flags.cylc7_back_compat:
                     msg = (
-                        'deprecated graph items were automatically upgraded '
+                        'graph items were automatically upgraded '
```

Comment:
IMO think the word deprecated is useful as part poking users to upgrade.

Reply:
The message is printed for obsoleted items as well as for deprecated ones.

I'll make the message more instructive.